release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2995,6 +2995,54 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* When upgrading GCP clusters that use a boot disk that is not compatible with UEFI, shielded VM support cannot be enabled. Previously, this prevented the creation of new machines. With this release, shielded VM support is disabled for disks that are known to be incompatible with UEFI. This primarily affects customers upgrading from {product-title} version 4.12 to 4.13 using the GCP marketplace images. (link:https://issues.redhat.com/browse/OCPBUGS-52495[*OCPBUGS-52495*]) 
+
+* Previously, node logs on the {product-title} web console did not close when you clicked outside the node logs menu. With this release, the node logs menu now closes when you click outside the node logs menu. (link:https://issues.redhat.com/browse/OCPBUGS-52490[*OCPBUGS-52490*]) 
+
+// SME 
+* Previously, when you logged on to the Developer Sandbox from the {product-title} web console, the web console ignored the path in the URL and navigated the user to the `all projects` view on the Developer Sandbox instead of the namespace detailed in the URL. With this release, a fix corrects this behavior to the error no longer exists. (link:https://issues.redhat.com/browse/OCPBUGS-52406[*OCPBUGS-52406*]) 
+
+* Previously, the `capturegroup` inline diff algorithm in the `cluster-compare` tool failed to match the source text in an object with the capturegroup` regular expression from a reference template. This issue existed if the source text had a similar structure to a regular expression. With this release, a fix to the `capturegroup` inline diff algorithm means that this matching issue no longer occurs.(link:https://issues.redhat.com/browse/OCPBUGS-51306[*OCPBUGS-51306*])
+
+* Previously, when running `oc-mirror` v2 on a continuous integration (CI) automation cycle and you viewed `oc-mirror` v2 logs on a non-TTY console, the output was missing progress information, because of an issue with the progress bar implementation. With this release, `oc-mirror` v2 now disables the progress bar implementation and uses plaintext logging instead for redirecting output so that the missing information no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-50996[*OCPBUGS-50996*])
+
+* Previously, when the cluster certificate authority (CA) bundle was updated with a custom CA bundle, a delay would occur for the change to reflect in the `insights-runtime-extractor` container. This issue occurred if the Insights Operator gathered data after the CA bundle is updated. The container might not have been restarted and the workload runtime data would not be gathered. With this release, a fix removes the delay so that this issue no longer occurs.(link:https://issues.redhat.com/browse/OCPBUGS-48790[*OCPBUGS-48790*])
```

Comment:
```suggestion
* Previously, if the Insights Operator gathered data after the cluster certificate authority (CA) bundle was updated with a custom CA bundle, a delay occurred for the change to be reflected in the `insights-runtime-extractor` container. As a consequence, the container might not be restarted, and the workload runtime data might not be gathered. With this release, a fix removes the delay so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-48790[*OCPBUGS-48790*])
```

Reply:
Note was removed from the advisory. Thanks for the review!